cprint/color_test.go

```diff
@@ -0,0 +1,105 @@
+package cprint
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"sync"
+	"testing"
+
+	"github.com/fatih/color"
+	"github.com/stretchr/testify/assert"
+)
+
+func captureOutput(f func()) string {
+	reader, writer, err := os.Pipe()
+	if err != nil {
+		panic(err)
+	}
+	color.Output = writer
+
+	out := make(chan string)
+	wg := new(sync.WaitGroup)
+	wg.Add(1)
+	go func() {
+		var buf bytes.Buffer
+		defer wg.Done()
+		io.Copy(&buf, reader)
+		out <- buf.String()
+	}()
+	defer wg.Wait()
+	f()
+	writer.Close()
+	return <-out
```

Comment:
Won't this work:

```go
var out bytes.Buffer
color.Output = out
f()
return out.String()
```

?

Reply:
Works!
It is amazing how my tunnel vision led me to this ugly monster.